UI/SDLGUI.cpp

```diff
@@ -286,7 +286,7 @@ void SDLGUI::SDLInit() {
         m_gl_context = SDL_GL_CreateContext(m_window);
     GLenum glew_status = glewInit();
 
-    if (!m_window || !m_gl_context || GLEW_OK != glew_status) {
+    if (!m_window || !m_gl_context /*|| GLEW_OK != glew_status*/) {
```

Comment:
Rather than ignoring the status completely, can you check for the specific error message that you want to ignore?

Reply:
Currently there is Glew still required during cmake config and sdl2-dev on Debian 11 still requires x11-dev:

Also during FreeOrion cmake config there is **without glew** config error(on my old pc)

```
-- Found ZLIB: /usr/lib/i386-linux-gnu/libz.so (found version "1.2.11")
-- Found SDL: /usr/lib/i386-linux-gnu/libSDL2.so (found version "2.0.14")
-- Found Freetype: /usr/lib/i386-linux-gnu/libfreetype.so (found version "2.10.4")
-- Found OpenGL: /usr/lib/i386-linux-gnu/libOpenGL.so
-- Found OpenAL: /usr/lib/i386-linux-gnu/libopenal.so
-- Found Ogg: /usr/lib/i386-linux-gnu/libogg.so
-- Found Vorbis: /usr/lib/i386-linux-gnu/libvorbis.so
-- Boost 1.69.0 found.
-- Found Boost components:
   filesystem;regex;system
CMake Error at /usr/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:165 (message):
  Could NOT find GLEW (missing: GLEW_INCLUDE_DIRS GLEW_LIBRARIES)
Call Stack (most recent call first):
  /usr/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:458 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.18/Modules/FindGLEW.cmake:207 (find_package_handle_standard_args)
  GG/CMakeLists.txt:69 (find_package)


-- Configuring incomplete, errors occurred!
```

Reply:
I suppose glew is used in code and we cannot make it optional.

Reply:
According [Wayland protocol on wiki](https://en.wikipedia.org/wiki/Wayland_(display_server_protocol)) Weston(example implementation of Wayland procotol(GNOME uses own implementation of Wayland protocol)) Weston uses [OpenGL ES](https://en.wikipedia.org/wiki/OpenGL_ES), which:

> For rendering, Weston can use [OpenGL ES](https://en.wikipedia.org/wiki/OpenGL_ES)[[60]](https://en.wikipedia.org/wiki/Wayland_(display_server_protocol)#cite_note-Paalanen_2012_EGL-60) or the pixman library to do [software rendering](https://en.wikipedia.org/wiki/Software_rendering).[[61]](https://en.wikipedia.org/wiki/Wayland_(display_server_protocol)#cite_note-Larabel_2013_pixman-61) The full OpenGL implementation is not used, because on most current systems, installing the full OpenGL libraries would also install [GLX](https://en.wikipedia.org/wiki/GLX) and other [X Window System](https://en.wikipedia.org/wiki/X_Window_System) support libraries as dependencies.

And OpenGL ES is not OpenGL 2.x FreeOrion requirements...

Reply:
> Rather than ignoring the status completely, can you check for the specific error message that you want to ignore?

Should work, will do. But I think the error was rather unspecific 'Unknown error'

Reply:
> during FreeOrion cmake config there is **without glew** config error(on my old pc)(with boost 1.74)

@hifron we need to load GL up, so we need something like GLEW; https://www.khronos.org/opengl/wiki/OpenGL_Loading_Library

Reply:
@agrrr3 seems maybe fixed on GLEW master, post on forum

Reply:
SDL 3 is needed to run on Wayland natively, on current SDL 2 its on XWayland and on obsolete SDL 1 some sdl-compatibility package is needed to run it...

Reply:
@hifron SDL2 is able to run native on wayland. SDL3 is in early alpha (and probably in flux), I am not aware anything has been released.